src/Menu/MenuItem.js

```diff
@@ -1,39 +1,66 @@
-/** @jsx jsx */
-import { jsx } from '@emotion/core';
-import { hideVisually } from 'polished';
+import React from 'react';
 import PropTypes from 'prop-types';
+import { hideVisually } from 'polished';
 
 import StyledMenuItem from './StyledMenuItem';
 import Text from '../Text';
 import Icon from '../Icon';
 
-const MenuItem = ({
-  selected,
-  baseColor,
-  id,
-  label,
-  name,
-  onSelect,
-  value,
-  focused
-}) => (
-  <StyledMenuItem
-    {...{
-      baseColor,
-      selected,
-      focused,
-      role: 'Menuitem',
-      'aria-selected': selected,
-      alignItems: 'center',
-      height: '40px',
-      onClick: () => onSelect(value)
-    }}
-  >
-    <input {...{ id, name, value, css: hideVisually(), type: 'hidden' }} />
-    <Icon name="status" mr="small" color={baseColor} fontSize="16px" />
-    <Text.span fontSize="size2">{label}</Text.span>
-  </StyledMenuItem>
-);
+export const ENTER_KEY = 'Enter';
+export const SPACEBAR = ' ';
```

Comment:
Move these to a helper file somewhere? Exporting them in this file seems weird.

Reply:
Yeah, I wasn't to sure how I felt about exporting this like this either. Any thoughts on what a good file / location might be?

Reply:
Let's start with `src/constants.js` and we go from there?